### PR TITLE
mkvmerge: add page

### PR DESCRIPTION
**Pull request by jtmlis**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/mkvmerge.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# mkvmerge
+
+> Merge and extract multimedia streams
+> More information: <https://github.com/topics/mkvmerge>.
```
Comment (Waples) [inline pages/linux/mkvmerge.md:4]:
```suggestion
> More information: <https://manned.org/mkvmerge>.
```

File: pages/linux/mkvmerge.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+- Print information about the source file:
+
+`mkvmerge --identify {{file.mkv}}`
```
Comment (Waples) [inline pages/linux/mkvmerge.md:8]:
```suggestion
`mkvmerge --identify {{path/to/file}}`
```

File: pages/linux/mkvmerge.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+- Extract the audio:
+
+`mkvextract tracks {{file.mkv}} {{1}}:{{audio.webm}}`
```
Comment (Waples) [inline pages/linux/mkvmerge.md:12]:
```suggestion
`mkvextract tracks {{path/to/file}} {{1}}:{{audio.webm}}`
```

File: pages/linux/mkvmerge.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Extract a subtitle track:
+
+`mkvextract tracks {{file.mkv}} {{3}}:{{subs.srt}}`
```
Comment (Waples) [inline pages/linux/mkvmerge.md:16]:
```suggestion
`mkvextract tracks {{path/to/file}} {{3}}:{{subs.srt}}`
```

**Review by Waples — CHANGES_REQUESTED** [review]

**Review by jtmlis — COMMENTED** [review]
reviewed and approved my mistakes

File: pages/linux/mkvmerge.md (hunk 3 above)
Comment (jtmlis) [reply pages/linux/mkvmerge.md:12]: thank you

**Comment by Waples** [comment]
@jtmlis no you didn't you just resolved the conversation :P

**Comment by jtmlis** [comment]
> @jtmlis no you didn't you just resolved the conversation :P

i reopened them.

File: pages/linux/mkvmerge.md (hunk 2 above)
Comment (navarroaxel) [reply pages/linux/mkvmerge.md:8]: If only files in `mkv` format are allowed as input then it should be:
```suggestion
`mkvmerge --identify {{path/to/file.mkv}}`
```

File: pages/linux/mkvmerge.md (hunk 1 above)
Comment (navarroaxel) [reply pages/linux/mkvmerge.md:4]: I think the main website is harder to find and it have a better explanation about the command.
```suggestion
> More information: <https://mkvtoolnix.download/doc/mkvmerge.html>.
```

Comment (marchersimon) [reply pages/linux/mkvmerge.md:4]: I think we should prefer the official page here: <https://mkvtoolnix.download/doc/mkvmerge.html>.

File: pages/linux/mkvmerge.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+- Print information about the source file:
+
+`mkvmerge --identify {{path/to/file.mkv}}`
+
+- Extract the audio:
+
+`mkvextract tracks {{path/to/file.mkv}} {{1}}:{{audio.webm}}`
+
+- Extract a subtitle track:
```
[the remark this answers is not in the document]
Comment (jtmlis) [reply pages/linux/mkvmerge.md:14]: thank you

**Comment by CleanMachine1** [comment]
Move the file to /pages/common/ now so that its not forgotten

File: pages/linux/mkvmerge.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+# mkvmerge
```
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/linux/mkvmerge.md:1]: @jtmlis delete this page :grimacing:

**Review by CleanMachine1 — APPROVED** [review]

**Review by bl-ue — COMMENTED** [review]
Tiny suggestion :)

**Review by bl-ue — APPROVED** [review]
Excellent — thank you @jtmlis! 🚀

Also, thank you for keeping up with the review process; it can be lengthy sometimes ;)

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**